internal/manifests/manifestutils/params.go

```diff
@@ -0,0 +1,22 @@
+package manifestutils
```

Comment:
Could it lead to some issues, that this params are in an `internal` path? E.g. if some other peace of software wants to import the CR?

Reply:
afaics the params are only used in `manifests.BuildAll` and all functions building manifests, for example `BuildCompactor` etc (which is located in `internal/`). Previously `Params` was defined in `internal/manifests/manifests.go`, I've moved it to `internal/manifests/manifestutils/params.go` (due to having a cyclic import otherwise).

The CR doesn't reference this struct, it's mainly for passing around params to the manifest-build-functions.